src/views/GovernanceReviewTeam/Actions/IssueLifecycleId.tsx

```diff
@@ -292,6 +294,18 @@ const IssueLifecycleId = () => {
                     name="nextSteps"
                   />
                 </FieldGroup>
+                <FieldGroup scrollElement="costBaseline">
```

Comment:
Do I need to handle error/error messaging for this FieldGroup? The field is optional so I don't think it's necessary but happy to add it if someone can identify a need.